### ISSUE TITLE
The instruction in the README doesn't work (julia-1.10.5)

### ISSUE THREAD

**Issue by uablrek** [natema/WorldDynamics.jl#210]
First:
```
cd /path/to/cloned/WorldDynamics.jl
julia
julia> ]add WorldDynamics
ERROR: ParseError:
# Error @ REPL[1]:1:1
]add WorldDynamics
╙ ── unexpected `]`
Stacktrace:
 [1] top-level scope
   @ none:1
```
That was easy to fix (but the instruction should be updated):
```
cd /path/to/cloned/WorldDynamics.jl
julia --project
```

Now the model can be used as instructed in the README (_only the first time_. please see below):
```
using WorldDynamics
```
but then:
```
World3.fig_7()
ERROR: ArgumentError: Any[dr₊pop(1900)] are either missing from the variable map or missing from the system's unknowns/parameters list.
(a *lot* of printouts omitted!)
```

### Julia version

I am using `julia-1.10.5` on Linux Ubuntu 24.04 LTS. I downloaded and unpack [julia-1.10.5-linux-x86_64.tar.gz](https://julialang-s3.julialang.org/bin/linux/x64/1.10/julia-1.10.5-linux-x86_64.tar.gz).

It is possible that the version I use (`julia-1.10.5`) incompatible, but then the supported Julia version should be documented. And/or preferably, the model should be updated for the latest Julia version.

### Subsequent attempts

When I exit Julia and try again:
```
using WorldDynamics
ERROR: The following 6 direct dependencies failed to precompile:
(a lot of printouts omitted!)
```
This can be "fixed" by clearing `~/.julia` (or $JULIA_DEPOT_PATH), but that should not be necessary.

**Comment by natema** [comment]
Thanks for flagging this, we'll look into it and let you know asap.

**Comment by aurorarossi** [comment]
When typing `] add WorldDynamics`, first type `]` and then `add WorldDynamics`.
Everything works with Julia 1.9.1. We are in the process of updating compatibility with other packages for newst Julia versions. Thank you very much!

**Comment by uablrek** [comment]
> When typing ] add WorldDynamics, first type ] and then add WorldDynamics

Yes, that worked, but most (all?) people cut/paste from an instruction.  And then you must also mention that you must *exit* the pkg-mode, which I by accident found out can be done with a backspace at the beginning of line (this is _not_ intuitive for most people).

**Comment by uablrek** [comment]
I backed down to julia-1.9.1 but got another error:
```
julia> World3.fig_7()
Error showing value of type PlotlyJS.SyncPlot:
ERROR: IOError: connect: connection refused (ECONNREFUSED)
Stacktrace:
  [1] wait_connected(x::Sockets.TCPSocket)
    @ Sockets /tmp/tmp/uablrek/julia/sys/share/julia/stdlib/v1.9/Sockets/src/Sockets.jl:528
(a lot of printouts omitted!)
```
It looks like "something else" must be started before this can be executed(?)

**Comment by aurorarossi** [comment]
> > When typing ] add WorldDynamics, first type ] and then add WorldDynamics
> 
> Yes, that worked, but most (all?) people cut/paste from an instruction. And then you must also mention that you must _exit_ the pkg-mode, which I by accident found out can be done with a backspace at the beginning of line (this is _not_ intuitive for most people).

Yes, it is true. We will update the README.md

**Comment by aurorarossi** [comment]
> I backed down to julia-1.9.1 but got another error:
> 
> ```
> julia> World3.fig_7()
> Error showing value of type PlotlyJS.SyncPlot:
> ERROR: IOError: connect: connection refused (ECONNREFUSED)
> Stacktrace:
>   [1] wait_connected(x::Sockets.TCPSocket)
>     @ Sockets /tmp/tmp/uablrek/julia/sys/share/julia/stdlib/v1.9/Sockets/src/Sockets.jl:528
> (a lot of printouts omitted!)
> ```
> 
> It looks like "something else" must be started before this can be executed(?)

It looks like there is a problem displaying the plot. I found a discussion with a similar problem on Ubuntu. Please check if you have these libraries installed:  https://github.com/JuliaGizmos/Blink.jl/issues/128

**Comment by uablrek** [comment]
I tried to set the BLINK_PORT variable to an unused port, as suggested in https://github.com/JuliaGizmos/Blink.jl/issues/128#issuecomment-401330727, but it didn't help (same error):
```
export BLINK_PORT=7789
```
But I think you are on the right track, the problem seem very similar. I am new to Julia so I don't know howto troubleshoot Blink.

**Comment by aurorarossi** [comment]
But what about the following packages:
    libgtk-3-0
    libxss1
    libgconf-2-4
    libnss3
    libasound2
    libx11-xcb1
Do you have them on Ubuntu?

**Comment by uablrek** [comment]
Seems like all except `libgconf-2-4` are there in one form or another. libgconf-2-4 seem to have gone out of time :disappointed: It seems like it _can_ be installed on Ubuntu from [ancient .deb packages](https://stackoverflow.com/questions/69880531/installing-libgconf-2-4-on-ubuntu), but I have got problems with upgrades after that in the past, so I don't want to do that.

I will test some newer Julia versions, I hope that will also include a Blink that can run on recent Linux distros. But if I can't get it to work, I'll leave this for now and wait for the model update. 

@aurorarossi Thank you very much for your help and patience. You took time to investigate and I really appreciate that.
